### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by qu4k**

### CODE REVIEW

File: src/daemon.ts [hunk 1]
```diff
@@ -23,87 +24,118 @@ export class Daemon implements AsyncIterable<DenonEvent> {
...
+      const command = commands[i];
+      let process = command.exe();
+      plog.debug(`starting process with pid ${process.pid}`);
+
+      if (i === commands.length - 1) {
+        plog.warning(`starting main \`${command.cmd.join(" ")}\``);
+        this.#processes[process.pid] = process;
+        this.monitor(process, command.options);
+        return command.options;
+      } else {
```
Comment (eliassjogreen) [inline src/daemon.ts:66]: No need for this else block

File: src/runner.ts [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
 export interface RunnerConfig extends ScriptOptions {
   scripts: Scripts;
 }
 
 const reDenoAction = new RegExp(/^(deno +\w+) *(.*)$/);
 const reCompact = new RegExp(
-  /^'(?:\\'|.)*?\.(ts|js)'|^"(?:\\"|.)*?\.(ts|js)"|^(?:\\\ |\S)+\.(ts|js)$/,
+  /^'(?:\\'|.)*?\.(ts|js)'|^"(?:\\"|.)*?\.(ts|js)"|^(?:\\ |\S)+\.(ts|js)$/,
 );
 const reCliCompact = new RegExp(/^(run|test|fmt) *(.*)$/);
```
Comment (eliassjogreen) [inline src/runner.ts:25]: We should probably add `lint` to this regex

File: src/daemon.ts (hunk 1 above)
Comment (eliassjogreen) [reply src/daemon.ts:66]: Or rather just remove the `else` keyword and brackets so we save those precious bytes or something lol

File: denon.ts [hunk 3]
```diff
@@ -14,27 +12,23 @@ import {
...
-  CompleteDenonConfig,
-  reConfig,
-} from "./src/config.ts";
+import { readConfig, CompleteDenonConfig, reConfig } from "./src/config.ts";
 import { parseArgs } from "./src/args.ts";
-import { setupLog } from "./src/log.ts";
+import log from "./src/log.ts";
+
+export const VERSION = "v2.3.0";
+export const BRANCH = "dev";
```
Comment (qu4k) [inline denon.ts:20]: Change branch

File: denon.ts [hunk 4]
```diff
@@ -14,27 +12,23 @@ import {
...
-  readConfig,
-  CompleteDenonConfig,
-  reConfig,
-} from "./src/config.ts";
+import { readConfig, CompleteDenonConfig, reConfig } from "./src/config.ts";
 import { parseArgs } from "./src/args.ts";
-import { setupLog } from "./src/log.ts";
+import log from "./src/log.ts";
+
+export const VERSION = "v2.3.0";
```
Comment (qu4k) [inline denon.ts:19]: Remove v

File: denon.ts [hunk 5]
```diff
@@ -117,7 +107,7 @@ if (import.meta.main) {
   }
 
   // show version number.
-  log.info(VERSION);
+  logger.info(`${VERSION}-${BRANCH}`);
```
Comment (qu4k) [inline denon.ts:110]: Add v

**Pull request merged**